myokit/_model_api.py

```diff
@@ -85,6 +85,7 @@ class ObjectWithMeta(object):
     Meta-data properties are all stored in a dict and should be string:string
     mappings.
     """
+
```

Comment:
```suggestion
```

Reply:
think PEP257 says we are supposed to put a blank line after multi-line docstrings: https://peps.python.org/pep-0257/#multi-line-docstrings? This was actually autopep8 that did this rather than me :)

Reply:
definitely after a class docstring, not sure about a function though, since the example they give doesn't have one!

Reply:
yea, autopep8 only does it for class docstrings:
```
Put a blank line between a class docstring and its first method declaration. (Enabled with E301.)
```

Reply:
Fair enough!
I figured there were some tools out there doing this :D
Feel free to revert, or not. It's not something I feel very strongly about